gradle.properties

```diff
@@ -1,7 +1,7 @@
 org.gradle.caching=true
 group=org.ballerinalang
-version=1.1.0-alpha5-SNAPSHOT
-ballerinaLangVersion=2.0.0-alpha5
-stdlibTaskVersion=1.2.0-alpha4
-stdlibLogVersion=1.1.0-alpha4
-stdlibIoVersion=0.6.0-alpha4
+version=1.1.0-alpha4-SNAPSHOT
```

Comment:
This should be `alpha5`. `alpha4` was the previously released version.

Reply:
Corrected in PR, https://github.com/ballerina-platform/module-ballerina-ftp/pull/168